coral/src/domain/team/team-types.ts

```diff
@@ -3,14 +3,14 @@
 // to be able to set "All teams" as a visible option
 // for the user, UUID makes sure we don't overlap with an
 
-import { KlawApiResponse } from "types/utils";
+import { KlawApiModel } from "types/utils";
 
 // actual team name in the future
```

Comment:
Not linked to this PR, but I believe these constants are not used any more, so probably should be deleted ^^

```
const ALL_TEAMS_VALUE = "f5ed03b4-c0da-4b18-a534-c7e9a13d1342";
const TEAM_NOT_INITIALIZED = "931bd061-fb50-4b92-ae49-b1e8004324d3";
```

Reply:
Will remove it, it's a bit related to the PR :D so 🧹

Reply:
https://github.com/Aiven-Open/klaw/pull/1970/commits/7b01b8f8dc7af3a22d0ebad700ae8ef1403435c1